packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1067,6 +1067,18 @@ export const MissingIndexForInternationalization = {
 	hint: (src: string) => `Create an index page (\`index.astro, index.md, etc.\`) in \`${src}\`.`,
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Some internationalization functions can't be exposed unless the default routing is disabled
+ */
+export const IncorrectStrategy = {
+	name: 'IncorrectStrategy',
```

Comment:
I would suggest using a name more specific to i18n, perhaps just `Incorrecti18nStrategy` would work